minecraft_servers/tryhub/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+  "server_name": "tryhub",
+  "nice_name": "TryHub",
+  "direct_ip": "tryhub.de",
+  "server_wildcards": [
+    "%.tryhub.de"
+  ],
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://tryhub.de",
+    "web_support": "https://tryhub.de/message-us",
+    "twitter": "TryHubDE",
+    "discord": "https://discord.gg/PFpuaMEK2E",
+    "teamspeak": "tryhub.de"
+  },
+  "discord": {
+    "server_id": 806169547408277564,
+    "rename_to_minecraft_name": true
```

Comment:
Please change to `false`, as this is the default value.

Reply:
fixed